lib/ansible/modules/cloud/cloudscale/cloudscale_server_group.py

```diff
@@ -38,6 +39,11 @@
       - Type of the server group.
     default: anti-affinity
     type: str
+  zone:
+    description:
+      - Zone in which the server resides (e.g. C(lgp1) or C(rma1)).
```

Comment:
Same as above `C()`, but as I can see this might do formatting? Also this is a server group not a server, maybe `The zone slug of the server group`, like you wrote below.